arangod/Cluster/MaintenanceFeature.cpp

```diff
@@ -308,7 +310,13 @@ void MaintenanceFeature::registerAction(std::shared_ptr<Action> action, bool exe
 
     if (!executeNow) {
       CONDITION_LOCKER(cLock, _actionRegistryCond);
-      _actionRegistryCond.signal();
+      _actionRegistryCond.broadcast();
```

Comment:
good catch!

Reply:
Thanks, I was switching in my mind between `signal` and `broadcast` multiple times but finally I understood the argument which I have put in the comment.